modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/MysqlSchemaCodegen.java

```diff
@@ -335,15 +334,15 @@ public void processIntegerTypeProperty(CodegenModel model, CodegenProperty prope
         String description = property.getDescription();
         String minimum = property.getMinimum();
         String maximum = property.getMaximum();
-        Boolean exclusiveMinimum = property.getExclusiveMinimum();
-        Boolean exclusiveMaximum = property.getIExclusiveMaximum();
+        boolean exclusiveMinimum = property.getExclusiveMinimum();
```

Comment:
Didn't know that Java scalars shouldn't start with capital. Maybe you need to describe it somewhere in contributor guide. When created my first PR I used `AbstractPhpCodegen` as reference and it filled up with `String` declarations. Or maybe we need some Java linter to avoid this style changes in future.

Reply:
This is called boxing in Java. It'll be caught by Sonar, we just have low stats on Sonar at the moment and don't make it known that we track there.

In the past couple of months, we've also introduced some quality checks in build which would warn on boxing. I think reviewers just need to be diligent on quality review.

That's not to say this is wrong. A `Boolean` in Java is an object that wraps a primitive `boolean`. Its usually frowned upon in performance critical paths, while in others it makes the most sense because of the true/false/null triple states.  It should flag as a warning in most IDEs for a line like this one as unnecessary boxing, because the method you're calling returns `boolean` and there's no possibility of that third null state which would represent "unset".